episodes/01-intro.md

```diff
@@ -26,52 +26,39 @@ The most widely used way to interact with personal computers is called a
 With a GUI, we give instructions by clicking a mouse and using menu-driven interactions.
 
 While the visual aid of a GUI makes it intuitive to learn,
-this way of delivering instructions to a computer scales very poorly.
+this way of interacting scales very poorly for some tasks.
+
+A **command-line interface** (CLI) allows users to to interact with computer by reading and writing text. It excels at making repetitive tasks automatic and fast.
+
+A **shell** is a particular program that lets you type commands.
+In this workshop, we will be using "Bash" which is the most popular Unix shell.
+Bash is often the default shell on Unix and in Unix-like tools for Windows.
+
+### Why use the shell?
+
 Imagine the following task:
 for a literature search, you have to copy the third line of one thousand text files in one thousand
 different directories and paste it into a single file.
+
 Using a GUI, you would not only be clicking at your desk for several hours,
 but you could potentially also commit an error in the process of completing this repetitive task.
-This is where we take advantage of the Unix shell.
-The Unix shell is both a **command-line interface** (CLI) and a scripting language,
-allowing such repetitive tasks to be done automatically and fast.
-With the proper commands, the shell can repeat tasks with or without some modification
-as many times as we want.
-Using the shell, the task in the literature example can be accomplished in seconds.
-
-### The Shell
-
-The shell is a program where users can type commands.
-With the shell, it's possible to invoke complicated programs like climate modeling software
-or simple commands that create an empty directory with only one line of code.
-The most popular Unix shell is Bash (the Bourne Again SHell ---
-so-called because it's derived from a shell written by Stephen Bourne).
-Bash is the default shell on most modern implementations of Unix and in most packages that provide
-Unix-like tools for Windows.
-Note that 'Git Bash' is a piece of software that enables Windows users to use a Bash like interface
-when interacting with Git.
+
+Using a CLI, you can write a series of commands that does this consistently and nearly instantly.
+
+The shell can be used for simple tasks like creating an empty folder
+and for launching (even complex) programs with a single command.
+In fact, some specialized tools and resources
+such as high-performance computing systems
+virtually require users to be familliar with the shell.
+Shell commands can be combined together and saved into reddproducible *scripts*
+that handle large volumes of data automatically.
```

Comment:
```suggestion
In fact, some tools and resources such as cloud computing systems
usually require users to be familiar with the shell.
Shell commands can be combined together and saved into reproducible *scripts*
that handle large volumes of data automatically.
```

Spelling corrections.  More users may find they log into a remote cloud system than
a remote HPC system.  Single board computers typically also run without GUIs to save
processing power for other tasks - maybe this is a better example?